client/components/videos-ui/video-modal.jsx

```diff
@@ -1,14 +1,11 @@
 import { useEffect } from 'react';
 import { BlankCanvas } from 'calypso/components/blank-canvas';
-import VideosUi from 'calypso/components/videos-ui';
+import VideosUi from 'calypso/components/videos-ui/index';
 import ModalFooterBar from 'calypso/components/videos-ui/modal-footer-bar';
 import ModalHeaderBar from 'calypso/components/videos-ui/modal-header-bar';
-import { COURSE_SLUGS } from 'calypso/data/courses';
 
-import './style.scss';
-
-const BloggingQuickStartModal = ( props ) => {
-	const { isVisible = false, onClose = () => {} } = props;
+const VideoModal = ( props ) => {
+	const { isVisible = false, onClose = () => {}, courseSlug = false } = props;
```

Comment:
Note: as long as you don't need the actual function parameter, you can destructure directly in the declaration:

```js
const VideoModal = ( { isVisible = false, onClose = () => {}, courseSlug = false } ) => {
```

Reply:
Also, you can omit the default type for `courseSlug` and rely on JS handling the falsy value.

```suggestion
	const { isVisible = false, onClose = () => {}, courseSlug } = props;
```